### PR TITLE
Updating all manual csvs

### PR DESCRIPTION
**Pull request by egekorkan**
I am updating the main and moving the changes to all implementations. Not sure if I will manage to finish today, thus draft for now

### CODE REVIEW

**Comment by egekorkan** [comment]
To have it as a list. 

To remove from each manual.csv:
- td-context-default-language-direction-heuristic
- td-context-default-language-direction-inference
- td-context-ns-multilanguage-text-direction-infer
- td-default-format-pop
- tm-placeholder-retyping
- tm-tmRef-overwrite
- well-known-operation-types-only

To add:
- privacy-auth-users-only
- privacy-centralized-ids
- privacy-essential-metadata-only
- privacy-immutable-id-as-property
- privacy-mutable-id-ownership
- privacy-mutable-identifiers
- privacy-td-pii
- privacy-temp-id-metadata
- sec-body-name-json-pointer-type
- sec-inj-no-intl-markup
- sec-inj-sanitize
- security-context-secure-fetch
- security-jsonld-expansion
- security-mutual-auth-td
- security-no-execution
- security-oauth-limits
- security-remote-context
- security-server-auth-td
- security-static-context
- security-update-contexts
- td-context-ns-td10-namespacev10
- td-security-extension
- td-text-direction-first-strong
- td-text-direction-language-tag
- thing-model-td-generation-processor-extends
- thing-model-td-generation-processor-forms
- thing-model-td-generation-processor-imports
- thing-model-td-generation-processor-placeholder
- thing-model-td-generation-processor-required
- thing-model-td-generation-processor-type
- tm-placeholder-replacement
- tm-ref-recursive-extensions
- tm-tmRef-overwrite-possibility
- tm-tmRef-overwrite-process
- tm-tmRequired-resolver

To possibly change order:
- td-default-AdditionalResponseContentType
- td-default-observable

To rename:
- tm-tmRef-2 -> tm-tmRef2

**Comment by egekorkan** [comment]
Removal and renaming was easy but adding new assertions is not. Each implementor needs to check if all the assertions above are in their manual.csv . Some of the assertions in the list have 16 hits in a search and some have 18. This means, not all manual.csvs are in sync

**Comment by mmccool** [comment]
Looks ok, but it looks like some of the new files have blank lines where an old assertion got deleted.  I don't *think* this will cause a problem with tooling, but it might (worst case, a tool might silently ignore things below the blank line) so it would probably be better if the blank lines were removed.  When I get a chance I will edit the PR to do that, then merge it, then update the TD testing results and implementation report.

File: data/input_2022/TD/Older/TUM/tum-PhilipsHUE.csv [hunk 1]
```diff
@@ -12,16 +12,13 @@
...
 "td-context-default-language-direction-independence","null","not testable with Assertion Tester"
-"td-context-default-language-direction-inference","null","not testable with Assertion Tester"
-"td-context-ns-multilanguage-text-direction-infer","null","not testable with Assertion Tester"
 "td-default-alg","null","not testable with Assertion Tester"
 "td-default-alg-pop","null","not testable with Assertion Tester"
 "td-default-contentType","null","not testable with Assertion Tester"
 "td-default-flow","null","not testable with Assertion Tester"
 "td-default-format","null","not testable with Assertion Tester"
-"td-default-format-pop","null","not testable with Assertion Tester"
+
```
Comment (mmccool) [inline data/input_2022/TD/Older/TUM/tum-PhilipsHUE.csv:21]:
```suggestion
```

File: data/input_2022/TD/Older/TUM/tum-microPythonBare.csv [hunk 2]
```diff
@@ -12,16 +12,13 @@
...
 "td-context-default-language-direction-independence","null","not testable with Assertion Tester"
-"td-context-default-language-direction-inference","null","not testable with Assertion Tester"
-"td-context-ns-multilanguage-text-direction-infer","null","not testable with Assertion Tester"
 "td-default-alg","null","not testable with Assertion Tester"
 "td-default-alg-pop","null","not testable with Assertion Tester"
 "td-default-contentType","null","not testable with Assertion Tester"
 "td-default-flow","null","not testable with Assertion Tester"
 "td-default-format","null","not testable with Assertion Tester"
-"td-default-format-pop","null","not testable with Assertion Tester"
+
```
Comment (mmccool) [inline data/input_2022/TD/Older/TUM/tum-microPythonBare.csv:21]:
```suggestion
```

File: data/input_2022/TD/Older/TUM/tum-mqttTest-retain.csv [hunk 3]
```diff
@@ -12,16 +12,13 @@
...
 "td-context-default-language-direction-independence","null","not testable with Assertion Tester"
-"td-context-default-language-direction-inference","null","not testable with Assertion Tester"
-"td-context-ns-multilanguage-text-direction-infer","null","not testable with Assertion Tester"
 "td-default-alg","null","not testable with Assertion Tester"
 "td-default-alg-pop","null","not testable with Assertion Tester"
 "td-default-contentType","null","not testable with Assertion Tester"
 "td-default-flow","null","not testable with Assertion Tester"
 "td-default-format","null","not testable with Assertion Tester"
-"td-default-format-pop","null","not testable with Assertion Tester"
+
```
Comment (mmccool) [inline data/input_2022/TD/Older/TUM/tum-mqttTest-retain.csv:21]:
```suggestion
```

File: data/input_2022/TD/Older/TUM/tum-mqttTest.csv [hunk 4]
```diff
@@ -12,16 +12,13 @@
...
 "td-context-default-language-direction-independence","null","not testable with Assertion Tester"
-"td-context-default-language-direction-inference","null","not testable with Assertion Tester"
-"td-context-ns-multilanguage-text-direction-infer","null","not testable with Assertion Tester"
 "td-default-alg","null","not testable with Assertion Tester"
 "td-default-alg-pop","null","not testable with Assertion Tester"
 "td-default-contentType","null","not testable with Assertion Tester"
 "td-default-flow","null","not testable with Assertion Tester"
 "td-default-format","null","not testable with Assertion Tester"
-"td-default-format-pop","null","not testable with Assertion Tester"
+
```
Comment (mmccool) [inline data/input_2022/TD/Older/TUM/tum-mqttTest.csv:21]:
```suggestion
```

File: data/input_2022/TD/Older/TUM/tum-pythonFlask.csv [hunk 5]
```diff
@@ -12,16 +12,13 @@
...
 "td-context-default-language-direction-independence","null","not testable with Assertion Tester"
-"td-context-default-language-direction-inference","null","not testable with Assertion Tester"
-"td-context-ns-multilanguage-text-direction-infer","null","not testable with Assertion Tester"
 "td-default-alg","null","not testable with Assertion Tester"
 "td-default-alg-pop","null","not testable with Assertion Tester"
 "td-default-contentType","null","not testable with Assertion Tester"
 "td-default-flow","null","not testable with Assertion Tester"
 "td-default-format","null","not testable with Assertion Tester"
-"td-default-format-pop","null","not testable with Assertion Tester"
+
```
Comment (mmccool) [inline data/input_2022/TD/Older/TUM/tum-pythonFlask.csv:21]:
```suggestion
```

File: data/input_2022/TD/Older/panasonic-client-browser/TDs/panasonic-client-browser.csv [hunk 6]
```diff
@@ -12,16 +12,13 @@
...
 "td-context-default-language-direction-independence","not-impl","language-direction is not supported"
-"td-context-default-language-direction-inference","not-impl","language-direction is not supported"
-"td-context-ns-multilanguage-text-direction-infer","not-impl","text-direction is not supported"
 "td-default-alg","pass",
 "td-default-alg-pop","not-impl","not supported"
 "td-default-contentType","pass",
 "td-default-flow","not-impl","not supported"
 "td-default-format","pass",
-"td-default-format-pop","not-impl","not supported"
+"","not-impl","not supported"
```
Comment (mmccool) [inline data/input_2022/TD/Older/panasonic-client-browser/TDs/panasonic-client-browser.csv:21]:
```suggestion
```

File: data/input_2022/TD/Older/panasonic-client-nodered/TDs/panasonic-client-nodered.csv [hunk 7]
```diff
@@ -12,16 +12,13 @@
...
 "td-context-default-language-direction-independence","not-impl","language-direction is not supported"
-"td-context-default-language-direction-inference","not-impl","language-direction is not supported"
-"td-context-ns-multilanguage-text-direction-infer","not-impl","text-direction is not supported"
 "td-default-alg","not-impl","not supported"
 "td-default-alg-pop","not-impl","not supported"
 "td-default-contentType","pass",
 "td-default-flow","pass",
 "td-default-format","not-impl","not supported"
-"td-default-format-pop","not-impl","not supported"
+
```
Comment (mmccool) [inline data/input_2022/TD/Older/panasonic-client-nodered/TDs/panasonic-client-nodered.csv:21]:
```suggestion
```

File: data/input_2022/TD/Older/panasonic-server-simulator/TDs/panasonic-server-simulator.csv [hunk 8]
```diff
@@ -12,16 +12,13 @@
...
 "td-context-default-language-direction-independence","not-impl","client test"
-"td-context-default-language-direction-inference","not-impl","client test"
-"td-context-ns-multilanguage-text-direction-infer","not-impl","client test"
 "td-default-alg","not-impl","client test"
 "td-default-alg-pop","not-impl","client test"
 "td-default-contentType","not-impl","client test"
 "td-default-flow","not-impl","client test"
 "td-default-format","not-impl","client test"
-"td-default-format-pop","not-impl","client test"
+
```
Comment (mmccool) [inline data/input_2022/TD/Older/panasonic-server-simulator/TDs/panasonic-server-simulator.csv:21]:
```suggestion
```

File: data/input_2022/TD/Results/huawei-cf-secure-psk.csv [hunk 9]
```diff
@@ -12,16 +12,13 @@
...
 "td-context-default-language-direction-independence","null","TD Producer only",
-"td-context-default-language-direction-inference","null","TD Producer only",
-"td-context-ns-multilanguage-text-direction-infer","null","TD Producer only",
 "td-default-alg","not-impl","",
 "td-default-alg-pop","not-impl","",
 "td-default-contentType","pass","",
 "td-default-flow","not-impl","",
 "td-default-format","not-impl","",
-"td-default-format-pop","not-impl","",
+,
```
Comment (mmccool) [inline data/input_2022/TD/Results/huawei-cf-secure-psk.csv:21]:
```suggestion
```

File: data/input_2022/TD/Results/panasonic-server-simulator.csv [hunk 10]
```diff
@@ -12,16 +12,13 @@
...
 "td-context-default-language-direction-independence","not-impl","client test",
-"td-context-default-language-direction-inference","not-impl","client test",
-"td-context-ns-multilanguage-text-direction-infer","not-impl","client test",
 "td-default-alg","not-impl","client test",
 "td-default-alg-pop","not-impl","client test",
 "td-default-contentType","not-impl","client test",
 "td-default-flow","not-impl","client test",
 "td-default-format","not-impl","client test",
-"td-default-format-pop","not-impl","client test",
+,
```
Comment (mmccool) [inline data/input_2022/TD/Results/panasonic-server-simulator.csv:21]:
```suggestion
```

**Review by mmccool — CHANGES_REQUESTED** [review]

**Comment by mmccool** [comment]
Made some suggested edits to get rid of "junk" lines.  @egekorkan can you review and if they look good, update the PR?

**Review by mmccool — APPROVED** [review]

**Comment by mmccool** [comment]
junk lines fixed, merging, will do any cleanup after attempting to re-run tooling.

**Pull request merged**